demo/build.gradle

```diff
@@ -39,15 +39,16 @@ android {
     }
 }
 
+// [START android_maps_utils_install_snippet]
 dependencies {
-    implementation project(':library')
```

Comment:
When building the demo app, this will always pull from Maven Central instead of building the library locally and using that dependency.  Is that the intent here?

Reply:
That's part of the intent. The other is so that this code snippet shows up in our documentation site so that whenever this version, this [page](https://developers.google.com/maps/documentation/android-sdk/utility/setup) is updated, too.

Reply:
Ok - could we leave `implementation project(':library')` in the file but commented out (basically inverting what was previously there)?

I know I frequently use the demo app to test changes to the library itself, and I'm guessing other contributors do too. Having the commented out text will help highlight that contributors shouldn't expect to see local library changes impact the demo app by default, and will also make it easy to manually toggle this when editing locally.

Reply:
Make sense. Will add back

Reply:
done in ffaf808